recipes/bson/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "bson" %}
+{% set version = "0.4.8" %}
+{% set sha256 = "6cea0806b4b85ab26f4241afb8e0f7e82232cdac52447bb3267b16fa975ef53a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```

Comment:
Well, it's the zeroth build on conda-forge, but not the zeroth on Anaconda.org.

But sure, it's not different in any meaningful way.